code/modules/chemistry/Reagents-Medical.dm

```diff
@@ -189,20 +189,14 @@ datum
 					M.changeStatus("stimulants", -4 SECONDS * mult)
 				if(M.hasStatus("recent_trauma")) // can be used to help fix recent trauma
 					M.changeStatus("recent_trauma", -2 SECONDS * mult)
-				if(holder.has_reagent(src.id,10)) // large doses progress somewhat faster than small ones
-					counter += mult
-					depletion_rate = 0.6 // depletes faster in large doses as well
-				else
-					depletion_rate = 0.3
 
 				switch(counter += 1 * mult)
-					if(1 to 7)
+					if(1 to 4)
 						if(probmult(7)) M.emote("yawn")
-					if(7 to 30)
+					if(4 to 15)
 						M.setStatus("drowsy", 40 SECONDS)
 						if(probmult(9)) M.emote(pick("smile","giggle","yawn"))
-					if(30 to INFINITY)
-						depletion_rate = 0.6
+					if(15 to INFINITY)
```

Comment:
Isn't this making it KO twice as fast now?

Reply:
This also removes the state at which it progresses twice as fast, it has the exact same minimum KO time as the current ether, since that one would progress at double speed(from 30 to 15) when above 10 units.